handshake/ecies.go

```diff
@@ -34,6 +34,9 @@ func ECIES(privKey *id.PrivKey, r *rand.Rand) Handshake {
 		remoteSecretKeyCh := make(chan []byte, 1)
 		defer close(remoteSecretKeyCh)
 
+		// A pointer to the pubKey contained in the privKey struct
+		localPubKeyPtr := privKey.PubKey()
```

Comment:
```suggestion
		localPubKey := privKey.PubKey()
```
In Go, we generally do not include type information in the names of variables.